ansible/vars/icds/icds_public.yml

```diff
@@ -61,6 +61,9 @@ postgres_users: "{{ secrets.POSTGRES_USERS }}"
 
 postgres_config:
   max_connections: 300
+  work_mem: '8MB'
+  shared_buffers: '512MB'
+  effective_cache_size: '8GB'
```

Comment:
Same as shared_buffers, I wanted to increase and test.  Until we reached 25% and 1/2, just being cautious.